src/praisonai/praisonai/ui/chat.py

```
@@ -423,18 +438,37 @@ def _get_or_create_agent(model_name: str, tools_enabled: bool = True):
-        instructions="""You are a helpful AI assistant with access to powerful tools.
+    # Add external agent tools
+    if selected_external_agents:
+        handler = _get_external_agents_handler()
+        if handler:
+            for agent_name in selected_external_agents:
+                try:
+                    integration = handler.get_integration(agent_name)
+                    if integration and integration.is_available:
+                        tools.append(integration.as_tool())
+                        logger.info(f"Added external agent tool: {agent_name}")
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)